### DISCUSSION TITLE
Why does the dashboard use the Hasura migrations API to make database modifications?

### DISCUSSION THREAD

**Discussion by lectrician1** [nhost/nhost#2398] [Q&A] [answered]
As a followup to https://github.com/nhost/nhost/issues/1220...

Why does the dashboard use the Hasura migrations API to make database modifications?

Wouldn't it be better if it used the normal [Hasura Schema API endpoint](https://hasura.io/docs/latest/api-reference/schema-api/index/) at `/v2/query`?  Then we wouldn't need to run `hasura console` locally to expose the migration API endpoint...

**Comment by dbarrosop** [comment, marked as answer]
We use the console because we need to be able to use the migration endpoint to create migrations.
